zh/tutorials/linux_sbus.md

```diff
@@ -25,7 +25,10 @@ GND ->S.bus GND
 
 ## 电路图
 ![](http://www.playuav.com/uploads/article/20160310/56cf0f65bb1f7437c1618041a30dc308.png)
-![](https://github.com/crossa/raspx4-sbus-rc-in/blob/master/sbusToSerial_small.jpg?raw=true)
+![](https://github.com/crossa/raspx4-sbus-rc-in/blob/master/sbusToSerial_small.jpg?raw=true)  
+
+## 安装实例
+![](https://raw.githubusercontent.com/crossa/raspx4-sbus-rc-in/master/example.jpeg)  
```

Comment:
Thanks for the example. Just a request: please identify the different parts and connectors so people can identify it with the schematic you shared.